pyvista/plotting/lights.py

```diff
@@ -32,31 +33,56 @@ class Light(vtkLight):
     Parameters
     ----------
     position : list or tuple, optional
-        The position of the light. The interpretation of the position depends
-        on the type of the light and whether the light has a transformation matrix.
-        See also the :py:attr:`position` property.
+        The position of the light. The interpretation of the position
+        depends on the type of the light and whether the light has a
+        transformation matrix.  See also the :py:attr:`position`
+        property.
 
     focal_point : list or tuple, optional
-        The focal point of the light. The interpretation of the focal point depends
-        on the type of the light and whether the light has a transformation matrix.
-        See also the :py:attr:`focal_point` property.
+        The focal point of the light. The interpretation of the focal
+        point depends on the type of the light and whether the light
+        has a transformation matrix.  See also the
+        :py:attr:`focal_point` property.
 
     color : string or 3-length sequence, optional
-        The color of the light. The ambient, diffuse and specular colors will
-        all be set to this color on creation.
+        The color of the light. The ambient, diffuse and specular
+        colors will all be set to this color on creation.
 
     light_type : string or int, optional
         The type of the light. If a string, one of ``'headlight'``,
-        ``'camera light'`` or ``'scene light'``. If an int, one of 1, 2 or 3,
-        respectively. The class constants ``Light.HEADLIGHT``, ``Light.CAMERA_LIGHT``
-        and ``Light.SCENE_LIGHT`` are also available, respectively.
+        ``'camera light'`` or ``'scene light'``. If an int, one of 1,
+        2 or 3, respectively. The class constants ``Light.HEADLIGHT``,
+        ``Light.CAMERA_LIGHT`` and ``Light.SCENE_LIGHT`` are also
+        available, respectively.
+
+        * A headlight is attached to the camera, looking at its focal
+        point along the axis of the camera.
+        * A camera light also moves with the camera, but it can occupy
+        a general position with respect to it.
+        * A scene light is stationary with respect to the scene, as it
+        does not follow the camera. This is the default.
+
+    intensity : float, optional
+        The brightness of the light (between 0 and 1).
+
+    cone_angle : float, optional
+        Cone angle of a positional light in degrees.
+
+    show_actor : bool, optional
+        Show an actor for a spotlight that depicts the geometry of the
+        beam.
+
+    positional : bool, optional
+        Set if the light is positional.
+
+        The default is a directional light, i.e. an infinitely distant
+        point source. A positional light with a cone angle of at least
+        90 degrees acts like a spherical point source. A positional
+        light with a cone angle that is less than 90 degrees is known
+        as a spotlight.
```

Comment:
I'd put the `positional` keyword before `cone_angle`and `show_actor`, because it defines what "positional" and "spotlight" mean.

Reply:
Done.